.github/scripts/trymerge.py

```diff
@@ -917,6 +918,26 @@ def prefix_with_github_url(suffix_str: str) -> str:
     return f"https://github.com/{suffix_str}"
 
 
+def check_for_sev(org: str, project: str, force: bool) -> None:
+    if force:
+        return
+    response = cast(
+        Dict[str, Any],
+        fetch_json(
+            "https://api.github.com/search/issues",
+            params={"q": f'repo:{org}/{project} is:open is:issue label:"ci: sev"'},
+        ),
+    )
+    if response["total_count"] != 0:
+        for item in response["items"]:
+            if "merge blocking" in item["body"].lower():
+                raise RuntimeError(
+                    "Not merging any PRs at the moment because there is a "
+                    + f"merge blocking ci: sev issue open at `{item['html_url']}`"
```

Comment:
```suggestion
                    "Not merging any PRs at the moment because there is a "
                    + "merge blocking https://github.com/pytorch/pytorch/labels/ci:%20sev issue open at: \n"
                    + f"{item['html_url']}"
```

You can actually reference labels in markdown and it should render them. (see [changelog update](https://github.blog/changelog/2022-02-03-reference-labels-in-markdown/))

https://github.com/pytorch/pytorch/labels/ci:%20sev